fili-core/src/main/java/com/yahoo/bard/webservice/table/resolver/AggregatableDimensionsMatcher.java

```diff
@@ -43,12 +43,12 @@ public AggregatableDimensionsMatcher(DataApiRequest request, TemplateDruidQuery
 
     @Override
     public boolean test(PhysicalTable table) {
-        Set<String> columnNames = TableUtils.getColumnNames(request, query.getInnermostQuery(), table);
+        Set<String> columnNames = TableUtils.getColumnNames(request, query.getInnermostQuery());
 
+        // If table contains non-agg dimensions, query must contain all these non-agg dimensions to use this table.
```

Comment:
This is a beautiful, perfect comment. Could we add it up in the class's JavaDoc though, so that it's more clear and easier to see what this class is requiring?